### PR TITLE
improve env sample and add envs documentation to readme

### PR DESCRIPTION
**Pull request by anamontiaga**
This PR includes improvements related to env variables:
- Rename env.sample to env.default (to unify with BE nomenclature).
- Add and comment new env variables to env.default.
- Add examples to env.default.
- Document env vars on ReadMe.
- Add env variables documentation to install info on Storybook.

### Issue
[#32](https://github.com/Vizzuality/front-end-scaffold/issues/32)

### CODE REVIEW

File: .env.default [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+# next-auth
+# ! do not forget to add a secret. NextAuth can handle without it in development mode,
+# ! but it won't in production!
+# ? https://next-auth.js.org/configuration/options#secret
+NEXTAUTH_SECRET=
+NEXTAUTH_URL=http://localhost:3000
+
+# API Url. Depending on the environment we will use different urls. It can be the same for the storybook and for the project.
+NEXT_PUBLIC_API_URL=https://api-url.com
+STORYBOOK_API_URL=https://api-url.com
```
Comment (andresgnlez) [inline .env.default:12]: `STORYBOOK_API_URL=$NEXT_PUBLIC_API_URL` <- passes the value as a reference so we don't have to write it twice and if the original one changes (`NEXT_PUBLIC_{WHATEVER}`), the other will update with new value.

image

_Source_: https://nextjs.org/docs/basic-features/environment-variables#loading-environment-variables

File: .env.default [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+# Mapbox token corresponding to the account used for this project. It can be the same for the storybook and for the project.
+NEXT_PUBLIC_MAPBOX_API_TOKEN=pk.eyJ1IjoibGV0aWNpYWhlY28iLCJhIjoiY2tzeGg3emczMHBzdTJ1cndkdXIxeGxlZyJ9.8yNE1jEYkE6tu-MAUzUfrA
+STORYBOOK_MAPBOX_API_TOKEN=pk.eyJ1IjoibGV0aWNpYWhlY28iLCJhIjoiY2tzeGg3emczMHBzdTJ1cndkdXIxeGxlZyJ9.8yNE1jEYkE6tu-MAUzUfrA
+
+# Google Analytics tracking ID. If you're working with an Google Analytics 4 property, you have a Measurement ID instead of a Tracking ID.
+NEXT_PUBLIC_GA_TRACKING_ID="UA-000000-2"
```
Comment (andresgnlez) [inline .env.default:19]: Quotes are not needed: `NEXT_PUBLIC_GA_TRACKING_ID=UA-000000-2` is fine

**Review by mbarrenechea — CHANGES_REQUESTED** [review]
Could you add all this to the storybook docs? you have a folder called `docs`. `install` file is where I think you should update it

File: docs/install.stories.mdx [hunk 3]
```diff
@@ -22,10 +22,36 @@ This process will install the Yarn binary and the dependencies of the projects l
...
 ```
-NEXTAUTH_URL=http://localhost:3000
+
+NEXTAUTH_SECRET=
+NEXTAUTH_URL=http://localhost:$PORT
+
+NEXT_PUBLIC_API_URL=https://api-url.com
+STORYBOOK_API_URL=$NEXT_PUBLIC_API_URL
+
+NEXT_PUBLIC_MAPBOX_API_TOKEN=pk.eyJ1IjoibGV0aWNpYWhlY28iLCJhIjoiY2tzeGg3emczMHBzdTJ1cndkdXIxeGxlZyJ9.8yNE1jEYkE6tu-MAUzUfrA
```
Comment (mbarrenechea) [inline docs/install.stories.mdx:50]: Could you remove the token please?

**Review by mbarrenechea — CHANGES_REQUESTED** [review]

**Pull request merged**